### DISCUSSION TITLE
Overview of commands and more examples e.g. crewai

### DISCUSSION THREAD

**Discussion by nic0711** [danielmiessler/Fabric#166] [Q&A]
Hi all

- In the videos @Daniel always uses commands that I can't find in the Diku. Are these system shortcuts on Daniel's computer or do they belong to Fabric? He explains some of them, but is there an overview?
Examples:  "pt", "p", "wc -w", "pbpaste", ...

- I am also very curious about an introduction / documentation of the crewai agents and what else is possible with them.

- I would also like to know more about the GUI variant and the server. somehow I don't really understand the descriptions so far ;)

Cheers

**Comment by xssdoctor** [comment]
Those are system shortcuts on his computer. However, I did include p.py, which is located in installer/client/cli. you can alias p=~/fabric/installer/client/cli/p.py. 

As far as agents, we are working on it. We included one agent so far as a proof of concept. 

As far as the gui go into installer/client/gui and run npm install electron, then npm start and it should run

**Reply by nic0711** [reply]
Great! Thank you!

**Reply by nic0711** [reply]
On the other hand...  it would of course be great if @danielmiessler could share his shortcuts - even if only for inspiration. There are a lot of them in the instructions, which is a bit confusing ;)

**Reply by rostcheck** [reply]
"wc -w" is a standard Unix shell command (word count, with count words argument). However I also would love to have @danielmiessler's pt alias/command

**Reply by DeltaCorvi** [reply]
It would be nice if pbpaste was integrated into Fabric. @danielmiessler uses it so often, and non-Mac people like me have to figure out 1) what it is, and 2) how to implement it in a non-Mac environment.

**Comment by black-backdoor** [comment]
`pbpaste` is a built-in command on macOS, and `p` is often used as an alias for it. Similarly, `pbcopy` is used to put text into the clipboard Network Chuck has a [video](https://www.youtube.com/watch?v=aMzdeZ8vGXQ) on how to set this up.
